samples/graph-bulk-meetings/csharp/EventMeeting/ClientApp/src/components/FileUpload.jsx

```diff
@@ -67,18 +67,18 @@ class App extends Component {
         }
     }
 
-    // Handler when user click on create button.
-    onCreateTeamsMeetingsButtonClick = (event) => {
+    // Handler when user click on create button.onCreateTeamsMeetingButtonClick
+    onCreateTeamsMeetingButtonClick = (event) => {
         microsoftTeams.app.getContext().then(async (context) => {
             var excelrows = [];
             for (var i = 1; i < this.state.rows.length; i++) {
                 var obj = {
-                    topicName: this.state.rows[i][0],
-                    trainerName: this.state.rows[i][1],
-                    startdate: this.state.rows[i][2],
-                    enddate: this.state.rows[i][3],
-                    timing: this.state.rows[i][4],
-                    participants: this.state.rows[i][5]
+                    TopicName: this.state.rows[i][0],
```

Comment:
Dont use camel case here

Reply:
i have to use because these parameters should match with model